From 40c2dd36c5da64919f69dd4f37de2ecf19f23058 Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Mon, 28 Jul 2014 13:58:49 +0000
Subject: text-step-bar: use correct utf-8 multibyte sequence for ■

For some reason in commit 113526408c5706c9efc7f43c3f8258354e762677
I commited the sequence \x25\x96\xa0 in place of the block character
that was there before.  I don't know where I got \x25 from, but it's
not right (and couldn't even be right since the high order bit isn't
set which is a requirement for the first byte in multi-byte utf-8
sequences).

This commit uses \xe2 instead which I got from doing:

$ echo ■  | xxd
0000000: e296 a00a                                ....
---
--- plymouth-0.9.0.orig/src/libply-splash-core/ply-text-step-bar.c
+++ plymouth-0.9.0/src/libply-splash-core/ply-text-step-bar.c
@@ -96,7 +96,7 @@ ply_text_step_bar_draw (ply_text_step_ba
         }
 
       /* U+25A0 BLACK SQUARE */
-      ply_text_display_write (step_bar->display, "%s", "\x25\x96\xa0");
+      ply_text_display_write (step_bar->display, "%s", "\xe2\x96\xa0");
       ply_text_display_write (step_bar->display, "%c", ' ');
     }
 
